src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --header-height: 60px;
+  --color-links: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  height: var(--header-height);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
The `header` element has `padding-left` and `padding-right` set to `50px`. Consider using CSS variables for these values to enhance maintainability and consistency.